### PR TITLE
issue 3 car moves with space

### PR DESCRIPTION
**Pull request by codywiseman**
startcar

### CODE REVIEW

File: driving game/main.js [hunk 1]
```diff
@@ -21,4 +25,15 @@ document.addEventListener('keydown', function (e) {
...
+  }
 });
+
+function startCar () {
+  $car.style.left = offSet[0] + 10 + 'px';
+  offSet[0] = $car.offsetLeft;
+  offSet[1] = $car.offsetTop;
+  carDirection.location = offSet;
+  intervalId = setInterval(startCar, 16)
+}
```
Comment (uzair-ashraf) [inline driving game/main.js:39]: This is using recursion, probably not on purpose.  log something in the console in this function to see what im talking about

**Review by uzair-ashraf — CHANGES_REQUESTED** [review]

File: driving game/main.js (hunk 1 above)
Comment (codywiseman) [reply driving game/main.js:39]: Does that just mean startCar is being called again every time the code on line 38 is run? So setInterval should be taken out of the function and instead get called on the keydown event?

Comment (uzair-ashraf) [reply driving game/main.js:39]: yes exactly

Comment (uzair-ashraf) [reply driving game/main.js:39]: every interval starts a new interval

Comment (codywiseman) [reply driving game/main.js]: startcar

File: driving game/main.js [hunk 2]
```diff
@@ -21,4 +23,16 @@ document.addEventListener('keydown', function (e) {
...
 });
+
+var positionX = 0;
+
+var intervalId = null;
+
+function startCar () {
+  positionX += 5;
+  $car.style.left = positionX + 'px';
+}
```
Comment (uzair-ashraf) [inline driving game/main.js:38]: Why have a postionX variable when you have an x property in your data model?
`carDirection.x+=5`

**Review by uzair-ashraf — CHANGES_REQUESTED** [review]

File: driving game/main.js (hunk 2 above)
Comment (codywiseman) [reply driving game/main.js]: Yea that makes total sense. I think I'm just over complicating it.

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**